privcount/traffic_model.py

```diff
@@ -276,8 +281,10 @@ def _get_inter_packet_delays(self, strm_start_ts, byte_events):
             delay = max(long(0), long(micros))
 
             # create a packet delay for each packet
-            while bw_bytes > 0:
+            split_packet_count = 0
+            while bw_bytes > 0 and split_packet_count < TrafficModel.MAX_SPLIT_PACKET_COUNT:
```

Comment:
I added some timing code and some warnings, and did some tests. I think we will run out of CPU when trying to process streams through the traffic model you emailed me.

For details, see the ticket:
https://github.com/privcount/privcount/issues/292#issuecomment-303667656